Makefile

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/make
+
+all: lint unit_test
+
+
+.PHONY: clean
+clean:
+	@rm -rf .tox
+
+.PHONY: apt_prereqs
+apt_prereqs:
+	@# Need tox, but don't install the apt version unless we have to (don't want to conflict with pip)
+	@which tox >/dev/null || (sudo apt-get install -y python-pip && sudo pip install tox)
+
+.PHONY: lint
+lint: apt_prereqs
+	@tox --notest
+	@PATH=.tox/py34/bin:.tox/py35/bin flake8 $(wildcard hooks reactive lib unit_tests tests)
+	@charm proof
+
+.PHONY: unit_test
+unit_test: apt_prereqs
+	@echo Starting tests...
+	tox
+
+.PHONY: fetch_resources
+fetch_resources:
```

Comment:
This is more for my sake, while I'm revving the charm testing different bins and preparing for a publish this script keeps me honest. I've also leveraged it in the README for users wishing to explore the functionality. (it wasn't originally worded this way)

its safe to ignore from a user perspective, but important from a release management perspective.